eng/pipelines/templates/jobs/vmr-build.yml

```
@@ -479,6 +479,12 @@ jobs:
         targetFolder=$(Build.StagingDirectory)/BuildLogs/
         mkdir -p ${targetFolder}
 
+        # Download rsync if using mariner
+        if grep -q "CBL-Mariner" /etc/os-release; then
```

In case the VM ever gets upgraded to Azure Linux 3.0, which is the next version after CBL Mariner 2.0, we should account for that.

```suggestion
        if grep -q -e "CBL-Mariner" -e "Azure Linux" /etc/os-release; then
```

---

Or
```suggestion
        if grep -q -E "CBL-Mariner|Azure Linux" /etc/os-release; then
```